target/linux/pistachio/base-files/etc/init.d/bt-csr

```diff
@@ -0,0 +1,215 @@
+#!/bin/sh /etc/rc.common
+#
+# Copyright 2016 by Imagination Technologies Limited and/or its affiliated group companies.
+#
+# All rights reserved.  No part of this software, either
+# material or conceptual may be copied or distributed,
+# transmitted, transcribed, stored in a retrieval system
+# or translated into any human or computer language in any
+# form by any means, electronic, mechanical, manual or
+# other-wise, or disclosed to the third parties without the
+# express written permission of Imagination Technologies
+# Limited, Home Park Estate, Kings Langley, Hertfordshire,
+# WD4 8LZ, U.K.
+
+START=80
+STOP=90
+
+source /lib/functions/mac.sh
+source /lib/functions/pistachio.sh
+source /lib/functions/system.sh
+
+BT_RST_N_GPIO=0
+MARDUK_BOARD="Marduk"
+
+board=$(pistachio_board_name)
+
+board_name=${board:0:${#MARDUK_BOARD}}
```

Comment:
Wasn't the point of changing to board name that it's always called Marduk anyway so this line isn't needed?

root@OpenWrt:/# cat /tmp/sysinfo/board_name
Marduk